src/BlazorUI/Demo/Client/Bit.BlazorUI.Demo.Client.Core/Pages/Components/Progress/Loading/BitLoadingDemo.razor.scss

```
@@ -16,3 +16,57 @@
+    padding: rem2(24px);
+    height: rem2(200px);
+    border-radius: rem2(4px);
+    flex-flow: column nowrap;
+    justify-content: center;
+    border: rem2(1px) solid var(--bit-clr-brd-sec);
+}
+
+// The classes the Style & Class example hands to the components themselves. They are written as
+// ::deep rules anchored on the plain wrappers above, because the scope attribute only ever lands on
+// elements written in this file - never on the root a component renders for a Class parameter.
+::deep .custom-class {
```

Stylelint (17.14.0)

[error] Unknown pseudo-element selector "::deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)